compiler/lib/flow.ml

```diff
@@ -429,13 +401,13 @@ let the_int info x =
   | Pc (Int c) -> Some c
   | Pc _ -> None
 
-let the_string_of ~target info x =
-  match the_const_of info ~target x with
+let the_string_of info x =
+  match the_const_of ~eq:(fun _ _ -> false) info x with
```

Comment:
Can you explain the `~eq:(fun _ _ -> false)` here and below ?

Reply:
This is used to specialize JavaScript primitives. We expect that a single string is directly passed to the primitive. So, I don't think we need to handle the case when there are several possible strings. But I can use `constant_equal` if you prefer.